awx/ui_next/src/components/JobCancelButton/JobCancelButton.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useCallback, useState } from 'react';
+import { t } from '@lingui/macro';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+import { Button, Tooltip } from '@patternfly/react-core';
+import { getJobModel } from '../../util/jobs';
+import useRequest, { useDismissableError } from '../../util/useRequest';
+import AlertModal from '../AlertModal';
+import ErrorDetail from '../ErrorDetail';
+
+function JobCancelButton({
+  job = {},
+  errorTitle,
+  title,
+  showIconButton,
+  errorMessage,
+}) {
+  const [isOpen, setIsOpen] = useState(false);
+  const { error: cancelError, request: cancelJob } = useRequest(
+    useCallback(async () => {
+      setIsOpen(false);
+      await getJobModel(job.type).cancel(job.id);
+    }, [job.id, job.type]),
+    {}
+  );
+  const { error, dismissError: dismissCancelError } = useDismissableError(
+    cancelError
+  );
+
+  return (
+    <>
+      <Tooltip content={title}>
+        {showIconButton ? (
+          <Button
+            aria-label={title}
+            ouiaId="cancel job"
+            onClick={setIsOpen}
+            variant="plain"
+          >
+            <MinusCircleIcon />
+          </Button>
+        ) : (
+          <Button
+            aria-label={title}
+            variant="secondary"
+            ouiaId="cancel job"
+            onClick={setIsOpen}
+          >{t`Cancel Sync`}</Button>
+        )}
+      </Tooltip>
```

Comment:
Toolti seems to be showing 2 arrows down.

<img width="296" alt="image" src="https://user-images.githubusercontent.com/9053044/117315709-ee654700-ae55-11eb-9168-896ae2fda46a.png">

Another one.

<img width="218" alt="image" src="https://user-images.githubusercontent.com/9053044/117315762-fcb36300-ae55-11eb-8cd3-af5a1ed14713.png">

Reply:
Good catch